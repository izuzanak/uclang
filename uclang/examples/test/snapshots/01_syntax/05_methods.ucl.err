 ---------------------------------------- 
Exception: ERROR: in file: "05_methods.ucl" on line: 91
      demo.non_static_private();

Cannot access private method Demo.non_static_private#0
 ---------------------------------------- 
