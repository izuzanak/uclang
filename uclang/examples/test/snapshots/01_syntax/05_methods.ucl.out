
=== All methods are accessible from its parent class ===========================

Demo.static_private()
Demo.static_public()
Demo.non_static_private()
Demo.non_static_public()

=== Call to private method throws an exception =================================


=== Public methods are accessible ==============================================

Demo.static_public()
Demo.static_public()
Demo.non_static_public()

=== Static methods can access static members ===================================

static_member: 0

=== Non-static methods can access all members ==================================

static_member: 0
non_static_member: 0

=== Tutorial end ===============================================================

